architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -0,0 +1,84 @@
+[[architecture-additional-concepts-ephemeral-storage]]
+= Ephemeral Local Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+*Note: this only applies if the ephemeral-storage tech preview in
+{product-title} 3.10 has been enabled.  This is disabled by default;
+to enable this feature, please see
+xref:../../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].*
+
+== Overview
+
+In addition to persistent storage, pods and containers may require
+ephemeral (transient) local storage for their operation.  The lifetime
+of this ephemeral storage does not extend beyond the life of the
+individual pod, and this ephemeral storage cannot be shared across
+pods.
+
+Prior to {product-title} 3.10, ephemeral local storage was exposed to
```

Comment:
Is there a difference between ephemeral local storage and ephemeral storage?

Reply:
Yes, I should use one or the other.  The code uses "ephemeralstorage" internally, but if you think ephemeral local storage would be a better term to use throughout, I will make that change globally.

Reply:
I think we should stick with what is used in the code.